### DISCUSSION TITLE
Harmony (ONE) and Ravencoin

### DISCUSSION THREAD

**Discussion by ILL-DIE-TRYING** [taoteh1221/Slideshow_Crypto_Ticker#31] [General]
I've been going through the exchanges and trying to find one that supports Harmony and Ravencoin. It looks to me like none do. Is there any way it could be added somehow?

Is there a way Coinmarketcap could be added with api key support maybe? I think there would be a much broader coin support if this were possible.

I haven't written any code for a while but I may be able to hack up something by looking at current code. I'm not a very informed GIT user but I would be willing to send my code to you if I can manage to make it work. Any advice on how to go about this?

**Comment by taoteh1221** [comment]
Hello. This app uses the "websockets API" protocol, which coinmarketcap didn't support last time I checked. Binance has RVN and ONE markets for BTC / USDT / BNB etc etc. This app already supports Binance out-of-the-box right now. Check out README.txt in the download, and config.js as well, for documentation on usage (adding coins and adjusting other settings). Holler if you get stuck.
